drop-db.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # ==========================================================================
-# Setup script for MYSQL Databse Instantiation
+# Setup script for MySQL database instantiation—used to drop the database.
```

Comment:
Should this just be `Setup script used to drop MySQL database`. It's not as though the script has any knowledge on how it's being used.